test_haystack/test_loading.py

```diff
@@ -350,3 +350,41 @@ def test_more_advanced_get_facet_field_name(self):
         self.assertEqual(self.ui._facet_fieldnames, {'bare_facet': 'bare_facet', 'title': 'title_facet', 'author': 'author_exact'})
         self.assertEqual(self.ui.get_facet_fieldname('title'), 'title_facet')
         self.assertEqual(self.ui.get_facet_fieldname('bare_facet'), 'bare_facet')
+
+    def test_collect_app_modules_indexes(self):
```

Comment:
If these functions get moved from `app_loading` to `loading`, then the tests can move too.